content/nginxaas-azure/getting-started/create-deployment/deploy-azure-portal.md

```diff
@@ -46,17 +48,44 @@ You can start the NGINXaaS deployment process by visiting the [Create NGINXaaS](
+   | Allow NGINX access to Virtual Network | Confirm that you allow:<br>- Registration of the NGINX provider to your Azure subscription.<br>- Delegation of the subnet to the NGINX provider.|
+   | IP address          | Set the IP address (public or private) that the service listens to for requests:<br><br>If you select a public IP address:<br>- Create a new public IP or use an existing one (for an existing one see the note below).<br>- Set the resource name for your public IP address.<br>Newly created public IPs are [zone-redundant in supported regions](https://learn.microsoft.com/en-us/azure/virtual-network/ip-services/public-ip-addresses#availability-zone). <br><br>If you select a private IP address:<br>- Provide a static IP address from the same subnet range set previously.   |
+   | Inbound port rules | Select `None` to disallow inbound access on any port, or choose to allow traffic from one of these common http(s) ports. <br><br> **Note:** This option is only available when specifying a new virtual network as part of the create workflow. If you select an existing virtual network which is associated with a subnet and Network Security Group (NSG), you will need to edit the Inbound security rules to add access for the specific ports you want to allow (for example, ports 80 and 443).|
+   | Apply default NGINX configuration | Confirm that you want your NGINXaaS deployment to be bootstrapped with a default NGINX configuration and a browsable splash page. |
    {{</bootstrap-table>}}
 
+   #### **Notes on subnets:**
+
+   - The minimum subnet size for NGINXaaS is `/27` and is enough for a single NGINXaaS deployment even at large scales.
+   - You can use an existing subnet; make sure that the subnet is delegated to `NGINX.NGINXPLUS/nginxDeployments` before creating a deployment in it. To delegate a subnet to an Azure service, see [Delegate a subnet to an Azure service](https://learn.microsoft.com/en-us/azure/virtual-network/manage-subnet-delegation?source=recommendations#delegate-a-subnet-to-an-azure-service).
+   - The subnet can contain other resources; ensure the subnet size is adequate to scale the NGINXaaS deployment.
+   - You can deploy more than one NGINXaas in the same subnet, however, every deployment in the subnet will share the address space (range of IP addresses that resources can use within the VNet), so ensure the subnet is adequately sized to scale the deployments. If you are more than one NGINXaaS deployment or other resources, a larger subnet, for example, a `/24` is recommended.
```

Comment:
```suggestion
   - You can deploy more than one NGINXaaS in the same subnet, however, every deployment in the subnet will share the subnet IP address space, so ensure the subnet is adequately sized to scale the deployments. If you have more than one NGINXaaS deployment or other resources, a larger subnet, for example, a `/24` is recommended.
```

I vaguely recall there being a blurb about things that might get impacted with an overloaded subnet. Eg: The abilitiy to receive deployment upgrades and not just scaling. Can we please add that here.

Reply:
I can't find any blurb that has been removed:
https://docs.nginx.com/nginxaas/azure/faq/#:~:text=How%20large%20should,%2C%20is%20recommended.

Reply:
Accepting the other suggestions on the original FAQ text